From e241905878cf3de78ad2f0ef4b16a2d12157884d Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Fri, 12 Jul 2024 09:56:01 +0000
Subject: [PATCH] Removing deprecated HALs from A15

These HALs have been deprecated in the latest API and
target requirements and hence removed.

Tests Done: Build and boot

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 .../device-specific/caas/compatibility_matrix.xml | 15 +++++++++++----
 1 file changed, 11 insertions(+), 4 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index 000faa8..6b86d07 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -1,5 +1,9 @@
 <compatibility-matrix version="1.0" type="device">
-    <hal format="hidl" optional="false">
+    <hal format="aidl">
+        <name>android.frameworks.sensorservice</name>
+        <fqname>ISensorManager/default</fqname>
+    </hal>
+    <!--hal format="hidl" optional="false">
         <name>android.frameworks.sensorservice</name>
         <version>1.0</version>
         <interface>
@@ -38,8 +42,11 @@
             <name>ITokenManager</name>
             <instance>default</instance>
         </interface>
-    </hal>
-    <!--vendor-ndk>
-        <version>34</version>
+    </hal-->
+    <!--system-sdk>
+        <version>VanillaIceCream</version>
+    </system-sdk>
+    <vendor-ndk>
+        <version>35</version>
     </vendor-ndk-->
 </compatibility-matrix>
-- 
2.45.2

